### DISCUSSION TITLE
Korrekte Auswahl der Gerätetypen bei Einweisungen & Gruppen

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#73] [Ideas]
In vielen Bereichen sind die Gerätetypen recht kryptisch und die User wissen oft nicht, welches Gerät nun das korrekte ist. User können das korrekte Gerät daher nur schwer für Einweisungen oder bei Einweisungsauswertungen identifizieren. Als eindeutige Identifikation gilt dabei doch häufig die Inventarnummer auf dem Gerät. 
Die Inventarnummer ist aktuell aber weder beim erstellen von Gruppen, noch in den Einweisungsauswertungen such-/filterbar.

Bsp. Befundungsmonitore der Fa. EIZO (https://www.eizo.com/products/radiforce/)

Die Monitore unterscheiden sich teilweise nur in der Größe. Von außen ist kein Typ/Modell erkennbar. Aktuell einzige Möglichkeit, Identifikation durch die Iventarnummer/QR auf dem Gerät.

**Comment by neongrau** [comment]
> [Gestern 21:25] Yves Rausch
> https://github.com/Samedis-care/samedis-care-community/discussions/73 können wir im backend die inventarnummer zur Suche von Gerätetypen heranziehen? das wäre dann ja ein Nr > Inventory > Typ 

Also man kann an jeder der Stellen praktisch schon immer nach Inventarnummer filtern. Das ist nur nicht kommuniziert/visualisiert und die Inventarnummer ist auch im Ergebnis nicht ersichtlich.
